### ISSUE TITLE
Bug: 

### ISSUE THREAD

**Issue by n02m4n** [qdm12/gluetun#1609]
### Is this urgent?

Yes

### Host OS

Debian GNU/Linux 11

### CPU arch

x86_64

### VPN service provider

Custom

### What are you using to run the container

Portainer

### What is the version of Gluetun

latest

### What's the problem 🤔

Hi, 

i get this error after deploy. 

"2023-05-26T10:55:40Z ERROR VPN settings: provider settings: server selection: for VPN service provider perfect privacy: the city specified is not valid: value is not one of the possible choices: value amsterdam,"

At my Config i assign "- SERVER_CITIES=Amsterdam", but in the Script at this check it is then apparently amsterdam.

I do not know why. My config ran without any problems until yesterday.

### Share your logs

```plain text
Running version latest built on 2023-05-25T12:12:44.932Z (commit a43973c)

🔧 Need help? https://github.com/qdm12/gluetun/discussions/new

🐛 Bug? https://github.com/qdm12/gluetun/issues/new

✨ New feature? https://github.com/qdm12/gluetun/issues/new

☕ Discussion? https://github.com/qdm12/gluetun/discussions/new

💻 Email? quentin.mcgaw@gmail.com

💰 Help me? https://www.paypal.me/qmcgaw https://github.com/sponsors/qdm12

2023-05-26T10:55:39Z INFO [routing] default route found: interface eth0, gateway 192.168.96.1, assigned IP 192.168.96.2 and family v4

2023-05-26T10:55:39Z INFO [routing] local ethernet link found: eth0

2023-05-26T10:55:39Z INFO [routing] local ipnet found: 192.168.96.0/20

2023-05-26T10:55:39Z INFO [firewall] enabling...

2023-05-26T10:55:39Z INFO [firewall] enabled successfully

2023-05-26T10:55:40Z INFO [storage] merging by most recent 13056 hardcoded servers and 13056 servers read from /gluetun/servers.json

2023-05-26T10:55:40Z ERROR VPN settings: provider settings: server selection: for VPN service provider perfect privacy: the city specified is not valid: value is not one of the possible choices: value amsterdam, choices available are Amsterdam, Basel, Belgrade, Berlin, Bucharest, Calais, Chicago, Copenhagen, Dallas, Erfurt, Frankfurt, Hamburg, Hongkong, Jerusalem, London, LosAngeles, Madrid, Malmoe, Manchester, Miami, Milan, Montreal, Moscow, NewYork, Nuremberg, Oslo, Paris, Prague, Reykjavik, Riga, Rotterdam, Singapore, Stockholm, Sydney, Tokyo, Vienna, Warsaw, Zurich

2023-05-26T10:55:40Z INFO Shutdown successful
```

### Share your configuration

```yml
version: "3"
services:
  gluetun:
    image: qmcgaw/gluetun
    #image: 15ky3/gluetun
    container_name: gluetun-mix
    cap_add:
      - NET_ADMIN
    devices:
      - /dev/net/tun:/dev/net/tun
    ports:
      - "8010:8000" # HTTP control server options
      - "8888:8888/tcp" # HTTP proxy
      - "8388:8388/tcp" # Shadowsocks
      - "8388:8388/udp" # Shadowsocks
    volumes:
      - /docker/gluetun/data:/gluetun
    restart: unless-stopped
    environment:
      - TZ="Europe/Berlin"
      - VPN_TYPE=openvpn
      - VPN_SERVICE_PROVIDER=perfect privacy
      - OPENVPN_USER=*****
      - OPENVPN_PASSWORD=*****
      - SERVER_CITIES=Amsterdam
      - HTTP_CONTROL_SERVER_ADDRESS:8000
      - HTTP_CONTROL_SERVER_LOG:on
```

**Comment by Deathproof76** [comment]
Got the same problems with Mullvad, tested V3.34 and V3.33.

**Comment by tasurel** [comment]
Got the same issue with surfshark and wireguard config

**Comment by vdrover** [comment]
Same here with NordVPN and `SERVER_REGIONS=Canada`. It seems the case of the regions is being dropped to `canada`.

**Comment by SimplicityGuy** [comment]
Dupe of #1599

**Comment by qdm12** [comment]
The latest image `qmcgaw/gluetun` was fixed a few minutes ago by 7e325715c7a24da73447b29cec94b207bd950745.
The regression was introduced 2 days ago with a43973c093662c2b5521eb2e9e3aea053584e96b which did compare text taking care about case sensitivity (gluetun lowercases everything inputted, but keeps servers data text with case sensitivity). My apologies for the delay fixing it, obviously, and thank you for reporting the issue 🎖️ 

Just in case you are not aware, you can use released docker images such as `qmcgaw/gluetun:v3` or `qmcgaw/gluetun:v3.34.1` which should be more stable than the latest image `qmcgaw/gluetun`. I always appreciate users using the latest image since this helps fix bugs before bringing them to a stable release which would end up unstable, so ideally keep using the latest image 😉
